### PR TITLE
Gabriel/recipe page

### PR DESCRIPTION
**Pull request by thatfoogabriel**
**What is the new behavior that this PR brings (describe or link to an issue)?**

**PR checklist (requester)**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Your code is documented

**PR checklist (reviewer)**

// TODO

**Other information**:
need to test bookmark, comment, and ratings connection to db, but logic is there in theory

### CODE REVIEW

File: src/components/circle_slider.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import CircularSlider from "react-circular-slider-svg";
+
+const CircleSlider = ({ onChange }) => {
```
Comment (blbergo) [inline src/components/circle_slider.tsx:4]: `export default function CircleSlider`

**Review by blbergo — APPROVED** [review]
lgtm

**Review by mgkram4 — COMMENTED** [review]
make sure to change the Merge into develop  rather than main, I did something similiar and need to revert. Other than that LGTM!

**Review by blbergo — APPROVED** [review]
lgtm

**Pull request merged**